src/index.d.ts

```diff
@@ -82,3 +82,4 @@ export const newStafiApp: SubstrateAppCreator
 export const newAlephZeroApp: SubstrateAppCreator
 export const newInterlayApp: SubstrateAppCreator
 export const newUniqueApp: SubstrateAppCreator
+export const newBifrostKusamaApp: SubstrateAppCreat
```

Comment:
It should be **SubstrateAppCreator**, you are missing 2 letters at the end 🙂

Reply:
Weird that CI is not complaining about that, we need to see what's happening there 😅

Reply:
fixed, so sorry. Thanks a lot